GREEDY/DataManagers/DataConverter.cs

```diff
@@ -1,122 +1,127 @@
+                    });
+                }
             }
-        }
-    
-        // TODO
-        // this doesnt belong in this class, can be moved to a static method, maybe an extension method
-        public XElement ListToXml(List<Item> items)
-        {
-            XElement xmlElements = new XElement("items",
-                items.Select(i => new XElement("item",
-                new XAttribute("Name", i.Name),
-                new XAttribute("Price", i.Price),
-                new XAttribute("Category", i.Category))));
-            return xmlElements;
-        }
 
-        // TODO
-        // this doesnt belong in this class, can be moved to a static method, maybe an extension method
-        public DataTable ListToDataTable(List<Item> items)
-        {
-            DataTable dataTable = new DataTable();
-            dataTable.Columns.Add("Name");
-            dataTable.Columns.Add("Price");
-            dataTable.Columns.Add("Category");
+            Console.WriteLine(receipt.Shop.Name);
+            Console.WriteLine(receipt.Date);
 
-            foreach (Item item in items)
+            //if no item was created. Need to create a massage for user
+            if (itemList.Count == 0)
             {
-                DataRow dr = dataTable.NewRow();
-                dr[0] = item.Name;
-                dr[1] = item.Price;
-                dr[2] = item.Category;
-                dataTable.Rows.Add(dr);
+                Console.WriteLine("Nepavyko nuskaityti cekio");
             }
-            return dataTable;
+
+            return itemList;
         }
+            //// TODO
+            //// this doesnt belong in this class, can be moved to a static method, maybe an extension method
+            //public XElement ListToXml(List<Item> items)
+            //{
+            //    XElement xmlElements = new XElement("items",
+            //        items.Select(i => new XElement("item",
+            //        new XAttribute("Name", i.Name),
+            //        new XAttribute("Price", i.Price),
+            //        new XAttribute("Category", i.Category))));
+            //    return xmlElements;
+            //}
+
+            //// TODO
+            //// this doesnt belong in this class, can be moved to a static method, maybe an extension method
+            //public DataTable ListToDataTable(List<Item> items)
+            //{
+            //    DataTable dataTable = new DataTable();
+            //    dataTable.Columns.Add("Name");
+            //    dataTable.Columns.Add("Price");
+            //    dataTable.Columns.Add("Category");
 
+            //    foreach (Item item in items)
+            //    {
+            //        DataRow dr = dataTable.NewRow();
+            //        dr[0] = item.Name;
+            //        dr[1] = item.Price;
+            //        dr[2] = item.Category;
+            //        dataTable.Rows.Add(dr);
+            //    }
+            //    return dataTable;
+            //}
```

Comment:
Commented code (will we still really need this). If yes maybe we can store it somewhere on github without merging it to master. (for example a copy of this branch)